### PR TITLE
Change to simple admin tool

### PR DESCRIPTION
**Pull request by kennyevil**

### CODE REVIEW

File: app/controllers/management_consultancy/admin/uploads_controller.rb [hunk 1]
```diff
@@ -14,14 +14,13 @@ def show
...
         @upload = ManagementConsultancy::Admin::Upload.new
-        @uploads_in_progress = ManagementConsultancy::Admin::Upload.in_review_or_in_progress?
       end
 
       def create
         @upload = ManagementConsultancy::Admin::Upload.new(upload_params)
 
         if @upload.save
-          ManagementConsultancy::DataScriptWorker.perform_async(@upload.id)
+          ManagementConsultancy::DataUploadWorker.perform_async(@upload.id)
```
Comment (iceraluk) [inline app/controllers/management_consultancy/admin/uploads_controller.rb]: are you still planning to use sidekiq?

Comment (kennyevil) [reply app/controllers/management_consultancy/admin/uploads_controller.rb]: Yes

Comment (iceraluk) [reply app/controllers/management_consultancy/admin/uploads_controller.rb]: ah ok. you will still have to wait for Gary to run the new terraform script changes which moves sidekiq to a separate container (I believe this is now planned for Monday as the Pipeline is still failing)

Comment (kennyevil) [reply app/controllers/management_consultancy/admin/uploads_controller.rb]: Ah, fair enough

Comment (kennyevil) [reply app/controllers/management_consultancy/admin/uploads_controller.rb]: @iceraluk Is this okay to merge?

Comment (iceraluk) [reply app/controllers/management_consultancy/admin/uploads_controller.rb]: @kennyevil it's ok to merge but this won't work just yet as sidekiq is not running anymore. We need to wait for Gary to run his terraform script with Sidekiq in a new container.

**Pull request merged**